src/components/CalendarMonthGrid.jsx

```diff
@@ -146,6 +146,9 @@ class CalendarMonthGrid extends React.Component {
       const withoutTransitionMonths = orientation === VERTICAL_SCROLLABLE;
       newMonths = getMonths(initialMonth, numberOfMonths, withoutTransitionMonths);
     }
+    
+    var momentLocale = moment.locale();
+    newMonths = newMonths.map(m => m.locale(momentLocale));
```

Comment:
Would it make sense to cache the locale in an instance variable and only do this if it has changed?

Also this is breaking some or our eslint rules (trailing spacings, no var, etc)